src/mon/MonCommands.h

```diff
@@ -447,7 +447,8 @@ COMMAND("osd dump " \
 	"print summary of OSD map", "osd", "r", "cli,rest")
 COMMAND("osd tree " \
 	"name=epoch,type=CephInt,range=0,req=false " \
-	"name=states,type=CephChoices,strings=up|down|in|out,n=N,req=false", \
+        "name=states,type=CephChoices,strings=up|down|in|out,n=N,req=false " \
+        "name=shadow,type=CephChoices,strings=--show-shadow,req=false", \
```

Comment:
@liewegas Can we keep this change? This is not the default behaviour of "osd tree" after all.
IMHO, the "crush tree" output is not very intuitive...